events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -0,0 +1,185 @@
+            <br>
+            <input type="button" id="selectedDifficulty" onclick="toggleMenu('difficultyDropdown')" class="gray" value="Any Difficulty   &#9660">
+            <ul class="dropdown-difficulty">
+                <div id="difficultyDropdown" class="dropdown-items">
+                    <li>Any Difficulty</li>
+                    <li>Easy</li>
+                    <li>Medium</li>
+                    <li>Hard</li>
+                </div>
+            </ul>
+            <br>
+            <input type="button" id="selectedCategory" onclick="toggleMenu('categoryDropdown')" class="gray" value="Any Category   &#9660">
+            <ul class="dropdown-category">
+                <div id="categoryDropdown" class="dropdown-items">
+                    <li value="9">General Knowledge</li>
+                    <li value="10">Books</li>
+                    <li value="11">Film</li>
+                    <li value="12">Music</li>
+                    <li value="13">Musicals and Theatres</li>
+                    <li value="14">Television</li>
+                    <li value="15">Video Games</li>
+                    <li value="16">Board Games</li>
+                    <li value="29">Comics</li>
+                    <li value="31">Japanese Anime and Manga</li>
+                    <li value="32">Cartoon and Animations</li>
+                    <li value="18">Computers</li>
+                    <li value="19">Mathematics</li>
+                    <li value="30">Gadgets</li>
+                    <li value="25">Art</li>
+                    <li value="26">Celebrities</li>
+                    <li value="27">Animals</li>
+                    <li value="28">Vehicles</li>
+                    <li value="20">Mythology</li>
+                    <li value="21">Sports</li>
+                    <li value="23">History</li>
+                    <li value="22">Geography</li>
+                    <li value="24">Politics</li>
+                </div>
+            </ul>
+            <br>
+        </div>
+        <p>
+        <input type="button" class="gray" id="newQuestion" value="New Question">
+        <p>
+        <input type="button" class="gray" id="showQuestion" value="Show This Question">
+        <input type="button" class="gray" id="showAnswers" value="Show The Answers">
+        <div id="question">
+            Your question will show here for preview. Make sure the question and answers do not have special 
+            characters and answers should not be longer than about 2/3 of this line.
+        </div>
+        <p>
+        <div id="answer">
+            Answer options will appear here.
+        </div>
+        <br>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+        <script>
+                function toggleMenu(menu) {
+                    document.getElementById(menu).classList.toggle("show");
+                }
+            
+                window.onclick = function(event) {
```

Comment:
Errant extra tab

Reply:
Also, why is this listener on the `window`? Is there a reason you don't have listeners on the individual elements?

Reply:
Rebecca wrote this, I never noticed anything weird happening so I haven't actually looked at most of this html code.

Reply:
OK, i'm not going to worry about it right now :)